### PR TITLE
CLN: remove empty parenteses on class creation

### PR DESCRIPTION
**Pull request by topper-123**
Minor clean-up: Some classes are created with a ``()`` which is not needed.

### CODE REVIEW

**Comment by jreback** [comment]
looks fine. can you add a rule which checks that we don't have this.

**Review by WillAyd — APPROVED** [review]

**Comment by topper-123** [comment]
I've added the rule. Could someone verify it works, I'm on Windows and can't run the script.

File: ci/code_checks.sh [hunk 1]
```diff
@@ -152,6 +152,10 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     invgrep -R --include="*.py" --include="*.pyx" -E "class\s\S*\(object\):" pandas scripts
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
+    MSG='Check for empty class parenteses' ; echo $MSG
+    invgrep -R --include="*.py" --include="*.pyx" -E "class\s\S*\(\):" pandas scripts
```
Comment (jschendel) [inline ci/code_checks.sh]: I think you can consolidate this with the previous check for inheriting from `object` with a regex pattern along the lines of `class\s\S*\((object)?\):`

**Comment by jschendel** [comment]
> Could someone verify it works, I'm on Windows and can't run the script.

I checked your original regex, and the one I suggested, and both look to work as intended.

FYI - installing git on windows often times will also give you git bash, which would allow you to test out these things (just replace `invgrep` with `grep`).

**Review by jschendel — APPROVED** [review]

**Comment by gfyoung** [comment]
FYI: The changes that were introduced into `code_checks.sh` had some pretty blatant typos and grammar errors.  I've corrected them in a separate commit.

**Pull request merged**